src/main/resources/reference.conf

```diff
@@ -44,5 +44,20 @@ resourceTypes =
           roleActions = ["launch_notebook"]
         }
       ]
+    },
+    {
+      name = "notebook-cluster"
+      actions = ["status", "connect", "sync", "delete", "read_policies"]
```

Comment:
i've made the actions in `LeoAuthProvider` match these

Reply:
cool - thanks 👍